src/objects/zcl_abapgit_sotr_handler.clas.testclasses.abap

```diff
@@ -0,0 +1,29 @@
+*"* use this source file for your ABAP unit test classes
+CLASS ltcl_sotr_handler DEFINITION FOR TESTING
+  DURATION SHORT
+  RISK LEVEL HARMLESS.
+  PRIVATE SECTION.
+    METHODS sotr_wda_0001 FOR TESTING.
+    METHODS sotr_cx_0002 FOR TESTING.
+ENDCLASS.
+
+CLASS ltcl_sotr_handler IMPLEMENTATION.
+  METHOD sotr_wda_0001.
+    DATA lt_sotr TYPE zif_abapgit_definitions=>ty_sotr_tt.
+    TRY.
+        lt_sotr = zcl_abapgit_sotr_handler=>read_sotr_wda( iv_object_name = 'ZWD_ABAPGIT_TEST_SOTR' ).
```

Comment:
I guess this unit test will fail if run on a system without ZWD_ABAPGIT_TEST_SORT? Which will be almost all systems?

Reply:
I thought the wd component is located in package $WDYN and part of this PR...

Reply:
Oops, I didn't expect Unit Tests relies on external objects.

Reply:
Alternatively we can use an existing WDR-Test-component which exists already in 7.02 - e.g. SALV_WD_TEST_TABLE_SIMPLE. Without the external component we can delete the Unit Test...

Reply:
@larshp Is it possible to use ABAP OpenSql Test Double Framework? https://help.sap.com/viewer/c238d694b825421f940829321ffa326a/1809.000/en-US/1432ca1fc7b547d493f691cdd09245ae.html

Reply:
no, it was introduced sometime long after 702

suggest just deleting this test case